pages/posts/index.jsx

```diff
@@ -8,22 +8,17 @@ import { promises as fs } from 'fs'
 import matter from 'gray-matter'
 import Header from '~components/Header'
 import Footer from '~components/Footer'
+import Tags from '~components/Tag'
+import { PostWrapper } from '~components/styled/Post.styled'
 
 const PostsPage = ({ posts }) => {
   return (
     <>
       <Header />
-      <div>
+      <PostWrapper>
         <h1>PostsPage</h1>
-
-        <ul>
-          {posts.map(({ slug, title }, index) => (
-            <li key={index}>
-              <Link href={`/blog/${slug}`}>{title}</Link>
-            </li>
-          ))}
-        </ul>
-      </div>
+        <Tags posts={posts} />
```

Comment:
Rename this Tag component to something else less specific i.e PostCardList since it's being used in both the posts and tag page

Reply:
Okay,wil work on all reviews